ci-operator/config/stackrox/stackrox/stackrox-stackrox-nightlies__ocp-4-17.yaml

```diff
@@ -1,89 +0,0 @@
-base_images:
-  ocp-4:
-    name: automation-flavors
-    namespace: stackrox
-    tag: openshift-4-0.10.13
-  ubi-minimal:
-    name: ubi-minimal
-    namespace: ocp
-    tag: "8"
-build_root:
-  image_stream_tag:
-    name: apollo-ci
-    namespace: stackrox
-    tag: stackrox-test-0.3.69
-resources:
-  '*':
-    requests:
-      cpu: 2000m
-      memory: 4000Mi
-test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
-tests:
-- as: qa-e2e-tests
-  postsubmit: true
-  steps:
-    env:
-      COLLECTION_METHOD: core_bpf
-      OCP_VERSION: ocp/candidate-4.17
```

Comment:
Just in case these tests were added intentionally, I think changing lines like this one to `OCP_VERSION: ocp-dev-preview/candidate-4.17` would work